app/views/bids/_form.html.erb

```diff
@@ -1,30 +0,0 @@
-<%= form_for bid_view_model.new_bid,
-  url: confirm_auction_bids_path(bid_view_model.id),
-  html: { class: 'bid-form' } do |f| %>
-  <div class="usa-width-one-whole">
-    <div class="usa-width-one-half">
-      <%= render partial: bid_view_model.highlighted_bid_info_partial,
-        locals: { bid_view_model: bid_view_model } %>
-    </div>
-
-    <div class="usa-width-one-half">
-      <h2 class="center"><label for="bid_amount">Your bid:</label></h2>
-      <%= f.text_field :amount %>
-    </div>
-  </div>
-
-  <p>
-    Note that all users who submit bids consent to the publication of their GitHub
```

Comment:
https://github.com/18F/micropurchase/blob/develop/app/views/logins/index.html.erb has it:

> None of the content in this GitHub repository is considered proprietary or procurement-sensitive information. Furthermore, all users who submit bids consent to the publication of their GitHub usernames, DUNS numbers, and bid amounts following the conclusion of the bidding process. Bidders' email addresses will not be disclosed and will only be used for micro-purchase-related communication on an as-needed basis.

Although, looks like we should get some clarification from @allalala about this line: "None of the content in this GitHub repository is considered proprietary". We might want to change that to say "website" or something else.